### PR TITLE
Expose `dt_storage_pool_log_gc_force_hardlink_rate` to users.

### PR DESCRIPTION
**Pull request by jiaqizho**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -168,6 +168,8 @@ delta_index_cache_size = 0
     ## 从 v5.4.0 引入，表示是否启用可自动扩展的线程池，这项功能可以显著提高 TiFlash 在高并发场景的 CPU 利用率。默认为 false。该功能为实验特性，不推荐在生产环境中开启。
     # enable_elastic_threadpool = false
 
+    ## 从 v5.4.0 引入，表示触发 TiFlash PageFile(log type) 进行 hardlink 时，最小的 valid rate 值。默认是不开启，如果需要 gc 时间过长，建议改成0.8以开启该功能。
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:171]:
```suggestion
    ## 从 v5.4.0 引入，表示触发 TiFlash PageFile(log type) 进行 hardlink 时最小的 valid rate 值。默认为 2，即不开启 PageFile 的 hardlink 功能。如果 GC 时间过长，建议将该值修改成 0.8 以开启 hardlink 功能。
```
下面的取值为 2，表示不开启么？

Comment (jiaqizho) [reply tiflash/tiflash-configuration.md:171]: 1. 是的，不开启。
2. valid rate = 单个文件内有效数据 / 单个文件内总数据量。

Comment (jiaqizho) [reply tiflash/tiflash-configuration.md]: commited.

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -168,6 +168,8 @@ delta_index_cache_size = 0
     ## 从 v5.4.0 引入，表示是否启用可自动扩展的线程池，这项功能可以显著提高 TiFlash 在高并发场景的 CPU 利用率。默认为 false。该功能为实验特性，不推荐在生产环境中开启。
     # enable_elastic_threadpool = false
 
+    ## 从 v5.4.0 引入，表示触发 TiFlash PageFile(log type) 进行 hardlink 时最小的 valid rate 值。默认为 2，即不开启 PageFile 的 hardlink 功能。如果 GC 时间过长，建议将该值修改成 0.8 以开启 hardlink 功能。
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:171]:
```suggestion
    ## 从 v5.4.0 引入，表示触发 TiFlash PageFile（log 类型） 进行 hardlink 时最小的 valid rate 值。默认为 2.0，即不开启 PageFile 的 hardlink 功能。如果 GC 时间过长，建议将该值修改成 0.8 以开启 hardlink 功能。
```

Comment (qiancai) [reply tiflash/tiflash-configuration.md:171]: @jiaqizho 如果取值小于 0.8 或者大于 0.8 有什么效果呢？

Comment (jiaqizho) [reply tiflash/tiflash-configuration.md:171]: 0.8 其实就是当 GC 发生时，有存在 valid rate 大于这个数(0.8)的 PageFile，就会被强制做 hardlink 。
如果值太低，可能会导致空间放大严重。如果值调太高，会使得触发条件严苛，从而触发 hardlink 次数变少，优化效果就比较小了。

Comment (qiancai) [reply tiflash/tiflash-configuration.md:171]: 请问 “dt_storage_pool_log_gc_force_hardlink_rate” 有 valid 的取值范围吗？如果有，可以在文档里说明下吗？

Comment (jiaqizho) [reply tiflash/tiflash-configuration.md:171]: 有的，在[0.1,1.0]。大于等于1.0即视为不生效。

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -168,6 +168,8 @@ delta_index_cache_size = 0
     ## 从 v5.4.0 引入，表示是否启用可自动扩展的线程池，这项功能可以显著提高 TiFlash 在高并发场景的 CPU 利用率。默认为 false。该功能为实验特性，不推荐在生产环境中开启。
     # enable_elastic_threadpool = false
 
+    ## 从 v5.4.0 引入，表示触发 TiFlash PageFile（log 类型） 进行 hardlink 时最小的 valid rate 值。默认为 2.0，即不开启 PageFile 的 hardlink 功能。如果 GC 时间过长，建议将该值修改成 0.8 以开启 hardlink 功能。
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:171]:
```suggestion
    ## 从 v5.4.0 引入，表示触发 TiFlash PageFile（log 类型） 进行 hardlink 时最小的 valid rate 值，取值范围为 [0.1,1.0]，取值越小，hardlink 越频繁；取值越大，触发 hardlink 条件越苛刻。默认为 2.0，即不开启 PageFile 的 hardlink 功能。如果 GC 时间过长，建议将该值修改成 0.8 以开启 hardlink 功能。
```

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request closed without merging**